packages/apps/composer-app/src/pages/DocumentPage.tsx

```diff
@@ -38,33 +39,31 @@ const nestedParagraphOutput = / +\n/g;
 
 const PureDocumentPage = observer(({ document }: { document: ComposerDocument }) => {
   const { t } = useTranslation('composer');
-
-  const editor = useComposerEditor({
-    document: document.content,
-    slots: { editor: { className: 'pbe-20' } }
-  });
+  const editorRef = useRef<TipTapEditor>(null);
```

Comment:
Ah I was wondering if I shoulda just used `ref`. Thanks for refactoring to this.